pushapkscript/docker.d/worker.yml

```diff
@@ -97,21 +102,34 @@ products:
           skip_check_same_locales: true
           skip_checks_fennec: true
           override_channel_model: "single_google_app"
-          single_app_config:
+          app:
             service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_REFERENCE_BROWSER" }
             credentials_file: { "$eval": "GOOGLE_CREDENTIALS_REFERENCE_BROWSER_PATH" }
             package_names:
               - "org.mozilla.reference.browser"
             certificate_alias: "reference-browser"
-
+        - product_names: [ "firefox-tv" ]
+          skip_check_multiple_locales: true
+          skip_check_same_locales: true
+          skip_checks_fennec: true
+          skip_check_signature: true
+          skip_check_ordered_version_codes: true
+          override_channel_model: "single_google_app"
+          apps:
```

Comment:
If the channel model for the dep `firefox-tv` is `single_google_app`, then this property should change from `apps` to `app` and it should be more google-specific, like [this one](https://github.com/mozilla-releng/scriptworker-scripts/pull/51/files#diff-3ab502ad0f8a21fc4b9a368b6cbcbd8eR105)

Reply:
Hm, dep `firefox-tv` has no `single_google_app` channel mode.

Reply:
I copy-pasted from puppet, hm.

Reply:
I think that puppet is wrong. You can confirm this by looking at [how `pushapkscript` parses the publish config](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/pushapkscript/pushapkscript/publish_config.py#L21) (note how it looks for an `app` property, not `apps`)

Reply:
I misunderstood, the dep `firefox-tv` config hasn't had `override_channel_model: "single_google_app"`, and should continue to not have that model. For some reason, I assumed that dep `firefox-tv` was like dep `reference-browser` and dep `fenix` in having that single dummy app.

Remove line 117 and this should be good to land!